### DISCUSSION TITLE
How can we make the log message for starting http server more eye-catching

### DISCUSSION THREAD

**Discussion by lellky** [playframework/playframework#11558] [Ideas]
I work in a project that uses Play framework. When it starts there are hundreds of rows of log output. It is really hard to see the log line where the Play server prints _Listening for HTTP on 127.0.0.1:9000"_

Could this be solved using colors and/or an ASCII character that is more eye-catching?

See this PR https://github.com/playframework/playframework/pull/11557

**Comment by szymond** [comment]
Have you considered changing port number? I think that way the line would be more noticeable.
